docs/src/main/asciidoc/jfr.adoc

```diff
@@ -0,0 +1,280 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= Using JDK Flight Recorder
+include::_attributes.adoc[]
+:categories: observability
+:summary: This guide explains how your Quarkus application can utilize JDK Flight Recorder to provide insight into itself.
+:topics: observability,jfr
+:extensions: io.quarkus:quarkus-jfr
+
+This guide explains how your Quarkus application can utilize https://openjdk.org/jeps/328[JDK Flight Recorder] (JFR) to provide
+insight into itself.
```

Comment:
Maybe add some more, like:
By adding this extension, a file names "something.jfr" will be stored in .../target (or .../build for Gradle) with the contents of the JFR event stream that is also enriched with Quarkus custom events.

Reply:
I have done the following based on your review

```
This guide explains how your Quarkus application can utilize https://openjdk.org/jeps/328[JDK Flight Recorder] (JFR) to provide
insight into itself.
JFR records various information from the Java standard API and JVM as events.
By adding this extension, you can add custom Quarkus events to JFR. This will help you solve problems in your application.

JFR can be preconfigured to dump a file, and when the application exits, JFR will output the file.
The file will contain the contents of the JFR event stream to which Quarkus custom events have also been added.
You can, of course, get this file at any time you want, even if your application exits unexpectedly.
```

Reply:
It looks good, can you please commit it?